### ISSUE TITLE
[Picker][TagPicker][Accessibilty] TagPicker container has role list but no list item child is present.

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#3385]
### Bug Report

- __Package version(s)__:  Latest
- __Browser and OS versions__: All browser
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No): Yes but not immediately

Requested priority: (Blocking, High, Normal, Low) : Normal

Products/sites affected: (if applicable): VSTS

#### Describe the issue:

TagPicker container has role list but no list item child is present.

#### Actual behavior:

Individual Tag does not have a role of ListItem

#### Expected behavior:

Individual Tag should be assigned a role of ListItem

### If applicable, please provide a codepen repro:

![image](https://user-images.githubusercontent.com/29277148/32766797-784c8f90-c936-11e7-84a7-87ab7c63e403.png)

**Comment by mywhere** [comment]
The issue is still exist for the default case which TagPicker has no value (selectedItem).
![image](https://user-images.githubusercontent.com/150653/53839975-bd271e80-3f4d-11e9-8249-e6d8a8a4d210.png)
